orttraining/orttraining/core/graph/pipeline_transformer.h

```diff
@@ -16,10 +16,22 @@ Status TransformGraphForPipeline(
     const std::unordered_set<std::string>& initializer_names_to_preserve,
     pipeline::PipelineTensorNames& pipeline_tensor_names);
 
-Status ApplyPipelinePartitionToMainGraph(
-    Graph& graph,
-    const std::vector<TrainingSession::TrainingConfiguration::CutInfo>& cut_info,
-    size_t pipeline_stage_id,
-    size_t num_pipeline_stage);
+// TODO(jufranc): when adapting this code to partition training graphs, add
+// a boolean is_training as parameter.
+Status ApplyPipelinePartitionToMainGraph(Graph& graph,
+    std::map<Node*, int>& op_to_rank,
```

Comment:
stage or rank?

Reply:
"rank" can be MPI rank or NCCL rank. Pipeline stage sounds clearer. :)

---
In reply to: [535180829](https://github.com/microsoft/onnxruntime/pull/5940#discussion_r535180829) [](ancestors = 535180829)

Reply:
I believe I've renamed them all, but do let me know if you find any rank that should be stage, please.